app/models/maintenance_stats/stats/bitbucket/commit_count_stats.rb

```diff
@@ -22,6 +23,11 @@ def count_up_commits(dataset, since)
           return unless dataset.present?
           dataset.select {|commit| Date.parse(commit["date"]) >= since}.size
         end
+
+        def latest_commit(dataset)
+          latest = dataset&.sort_by{|commit| Date.parse(commit["date"])}&.reverse&.first
```

Comment:
`reverse.first` == `last`?

Reply:
What format is `date` in - it'd be nice to not have to parse it.

Reply:
Will sorting work without Date parsing?

Reply:
Dates are formatted like this: `2017-02-17T16:25:10+00:00`. I think thats the 8601 standard format.